### ISSUE TITLE
findDOMNode is deprecated in StrictMode < Button / >

### ISSUE THREAD

**Issue by uinstinct** [Semantic-Org/Semantic-UI-React#4050]
## Bug Report
I was having this problem : 
`Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of RefFindNode which is inside StrictMode.` while working with semantic-ui-react

I would have had use _refForwarding_. But, now the problem is I have used Button on many different components.

### Steps
This thing thing can be reproduced easily in any when we are using the **Button** component.
```js
// App.js

import React from "react";

import "semantic-ui-css/semantic.min.css";
import { Button } from "semantic-ui-react";

export default function App() {
  return (
    <div className="App">
      <Button>Click Me</Button>
    </div>
  );
}
```

### Result

![Button Ref Problem](https://user-images.githubusercontent.com/61635505/91467926-e0b2f700-e8ae-11ea-8e2a-bc85827c0fb0.png)

### Version
"semantic-ui-react": "^1.2.0"
"semantic-ui-css": "^2.4.1"

### Testcase
**I have added a codesandbox url to see this problem easily:**

https://codesandbox.io/s/semantic-ui-react-button-issue-d6ipt?file=/src/App.js

**Comment by layershifter** [comment]
To avoid `findDOMNode()` we need to use `React.forwardRef()` internally. Duplicate for #3819.
Currently it's a warning that does not block any usage.

**Comment by uinstinct** [comment]
> 
> 
> To avoid `findDOMNode()` we need to use `React.forwardRef()` internally. Duplicate for #3819.
> Currently it's a warning that does not block any usage.

@layershifter But, it can be painful to write `forwardRef()` each time keeping in mind **I have used the Button Component multiple times and in multiple ways.** 
It would be very nice if the semantic-ui-react team could find a permenant solution to this problem. 🤔

**Comment by layershifter** [comment]
@uinstinct as consumer you are not forced to write `.forwardRef()`.

This warning comes from `Button` component that uses `Ref` & `RefFindNode` inside of it:

https://github.com/Semantic-Org/Semantic-UI-React/blob/c3640e4b9527a932661af4f786e9c03c726a708e/src/elements/Button/Button.js#L142
https://github.com/Semantic-Org/Semantic-UI-React/blob/c3640e4b9527a932661af4f786e9c03c726a708e/src/elements/Button/Button.js#L162

To remove the warning we need to remove usages of `Ref` component, to do this we need support ref forwarding on our components which is not currently supported.

For now, you can ignore this warning.

**Comment by uinstinct** [comment]
Sure, it will be very nice for the semantic-ui-react component are updated soon because we rely on it for doing for projects.

**Comment by hernandez-jordan** [comment]
Is there any update on this issue?

**Comment by ogradye** [comment]
could we get this on a milestone?

**Comment by layershifter** [comment]
This will be fixed by #4233.

**Comment by tennessine** [comment]
At least, semantic-ui is still in active development.

**Comment by panoxdesign** [comment]
When will this version be released?

**Comment by karlkras** [comment]
Why was this closed?

**Comment by layershifter** [comment]
> Why was this closed?

Because it's a duplicate of another issue that tracks the problem. This will be fixed in v3, you can track the process in #4233.

Thanks.
